content/specs/versions3.md

```diff
@@ -0,0 +1,148 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: macOS build machine specification (Xcode 12.5+)
+aliases:
+  - '../releases-and-versions/versions3'
+weight: 1
```

Comment:
weight: 3

Reply:
why 3? as I understand that would make it after xcode-11 specs and we will have them in order "xcode-12", "xcode-11", "xcode-12.5". I think we should leave 1 here and shift the others to be below this one so the order will be "xcode-12.5", "xcode-12", "xcode-11".

Reply:
true, agree